src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -77,7 +104,7 @@ public void Dispose()
             _parametersCache.Dispose();
         }
 
-        private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext, bool useAppContext)
+        private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appContext, bool useAppContext, bool ignoreVariant, CancellationToken cancellationToken)
```

Comment:
I'm not a fan of a the new parameter. It changes the method too much, by conditionally adding what I'd consider to be business logic worth it's own encapsulation. I'd recommend instead for those who need to do extra post-processing due to variant to do so at call-site.